include/OpenSpaceToolkit/Physics/Environment/Atmospheric/Earth.hpp

```diff
@@ -113,8 +137,13 @@ class Earth : public Model
     /// @param              [in] aLLA A LLA
     /// @param              [in] anInstant An Instant
     /// @return             Atmospheric density value [kg.m^-3]
+
     Real getDensityAt(const LLA& aLLA, const Instant& anInstant) const;
 
+    static constexpr double defaultF107ConstantValue = 150.0;   // 10⁻²² W⋅m⁻²⋅Hz⁻¹
```

Comment:
nit: defaults in Physics usually use header defines rather than `constexpr`
https://github.com/open-space-collective/open-space-toolkit-physics/blob/main/include/OpenSpaceToolkit/Physics/Time/Instant.hpp#L29

Reply:
I checked real quick with chatgpt, apparently
"
With this definition, every occurrence of DEFAULT_F107_CONSTANT_VALUE in your code will be replaced with 150.0 during the preprocessing stage. This is a simple textual substitution and doesn't have the type safety or scoping of true C++ constants.
For a more C++-style approach to defining constants with type safety, it's recommended to use const variables or, in C++11 and later, constexpr variables
" 🤷 Lmk but I'll keep it as is for now and we can update batch on the repo later